packages/notification-services-controller/CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.10.1]
+
+### Changed
+
+- update the types described in `types/on-chain-notification/schema` and `types/on-chain-notification/on-chain-notification` ([#4818](https://github.com/MetaMask/core/pull/4818))
+- update the mock data according to the new types in `mock-raw-notifications` ([#4818](https://github.com/MetaMask/core/pull/4818))
```

Comment:
This appears to be talking about test data that is not published with the package. These changelogs are written for people using this package, not for contributors _to_ the package, so things like this should be omitted

```suggestion
```

Reply:
These mocks are accessible and used on clients.

We export these and can be used via subpath exports/imports.

```
import { ... } from '@metamask/notification-services-controller/notification-services/mocks`
```

Reply:
Oh I see, interesting, nevermind then!